tests/parser/features/test_gas.py

```diff
@@ -11,8 +11,9 @@ def foo() -> int128:
 
     c = get_contract_with_gas_estimation(gas_call)
 
-    assert c.foo(startgas=50000) < 50000
-    assert c.foo(startgas=50000) > 25000
+    assert c.foo(call={"gas": 50000}) < 50000
+    assert c.foo(call={"gas": 50000}) > 25000
+
     print('Passed gas test')
```

Comment:
total nitpick on my part, but converting these `print` statements to use `logging` allows you to get the same output in your terminal while still allowing that output to be easily silenced, or even tuned to only show a subset by logger or by log level.